usaspending_api/disaster/tests/integration/test_federal_account_award.py

```diff
@@ -39,16 +42,14 @@ def test_federal_account_award_success(client, generic_account_data, monkeypatch
     resp = helpers.post_for_spending_endpoint(client, url, def_codes=["M", "L", "N", "O"], spending_type="award")
     expected_results = [
         {
+            "id": 21,
+            "code": "000-0000",
+            "description": "gifts",
+            "award_count": 4,
+            "obligation": 304.0,
+            "outlay": 667.0,
+            "total_budgetary_resources": None,
             "children": [
-                {
```

Comment:
If the order changed, does that mean the sorting behavior is different between the original and new?

Reply:
If I have time I'll investigate. The original sorted always added a secondary sort on an id field to provide deterministic inverse asc/desc ordering. But I would assume the default sort field to be obligation or outlay, which appears broken in these results

Reply:
The default sort behavior is to sort on descending obligated amount. I believe the previous behavior was to default to the id field, but this can easily be changed.